AppCenter/AppCenter/Internals/Util/MSConstants+Internal.h

```diff
@@ -152,3 +152,13 @@ static const long kMSMaximumCommonSchemaLogSizeInBytes = 2 * 1024 * 1024;
  * Suffix for One Collector group ID.
  */
 static NSString *const kMSOneCollectorGroupIdSuffix = @"/one";
+
+/**
+ * Typed properties.
```

Comment:
This should be part of the "MSACModelConstants". Also I am not a big fan of the existing convention but we should be consistent until we decide to change the convention.

Reply:
Can move them. They are used, though, by `MSEventLog` - the conversion logic must know the names of these and we don't want to hard code them twice. But we can move them to `MSACModelConstants`.